app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def home
+    @movies= Movie.all
```

Comment:
Small nitpick for code readability: leave a space between the instance variable and the equals sign, like:
```ruby
@movies = Movie.all
```

If you guys haven't seen it yet, [this style guide](https://github.com/bbatsov/ruby-style-guide) is pretty standard.